INVIOLABLES.md

```diff
@@ -1,4 +1,4 @@
-# The Inviolable Principles of Nuttx
+# The Inviolable Principles of NuttX
```

Comment:
Since it is the first heading we should get in the habit of calling the project by the full name `Apache NuttX` in the first reference.